projects/packages/connection/legacy/class-jetpack-signature.php

```diff
@@ -93,9 +93,9 @@ public function sign_current_request( $override = array() ) {
 			// Convert the $_POST to the body, if the body was empty. This is how arrays are hashed
 			// and encoded on the Jetpack side.
 			if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-				// phpcs:ignore WordPress.Security.NonceVerification.Missing
+				// phpcs:ignore WordPress.Security.NonceVerification.Missing -- Used to verify a cryptographic signature of the post data.
```

Comment:
The code in this function only generates the signature. The only caller does use it to verify against a passed-in signature though.
```suggestion
				// phpcs:ignore WordPress.Security.NonceVerification.Missing -- Used to generate a cryptographic signature of the post data.
```